test/models/multiple-results-event.tests.js

```diff
@@ -0,0 +1,427 @@
+    });
+
+    it('It should throw if resultSetStartTime is null', () => {
+      input.resultSetStartTime = null;
+      assert.throws(() => new MultipleResultsEvent(input), Error, "resultSetStartTime must be a Number");
+    });
+
+    it('It should throw if resultSetStartTime is undefined', () => {
+      input.resultSetStartTime = undefined;
+      assert.throws(() => new MultipleResultsEvent(input), Error, "resultSetStartTime must be a Number");
+    });
+
+    it('It should throw if resultSetStartTime is number', () => {
+      input.resultSetStartTime = "12";
+      assert.throws(() => new MultipleResultsEvent(input), Error, "resultSetStartTime must be a Number");
+    });
+
+    it('It should throw if resultSetStartTime is array', () => {
+      input.resultSetStartTime = [2];
+      assert.throws(() => new MultipleResultsEvent(input), Error, "resultSetStartTime must be a Number");
+    });
+
+    it('It should throw if resultSetStartTime is object', () => {
+      input.resultSetStartTime = {id:2};
+      assert.throws(() => new MultipleResultsEvent(input), Error, "resultSetStartTime must be a Number");
+    });
+
+    it('It should throw if betStartTime is null', () => {
+      input.betStartTime = null;
+      assert.throws(() => new MultipleResultsEvent(input), Error, "betStartTime must be a Number");
+    });
+
+    it('It should throw if betStartTime is number', () => {
+      input.betStartTime = "12";
+      assert.throws(() => new MultipleResultsEvent(input), Error, "betStartTime must be a Number");
+    });
+
+    it('It should throw if betStartTime is array', () => {
+      input.betStartTime = [2];
+      assert.throws(() => new MultipleResultsEvent(input), Error, "betStartTime must be a Number");
+    });
+
+    it('It should throw if betStartTime is object', () => {
+      input.betStartTime = {id:2};
+      assert.throws(() => new MultipleResultsEvent(input), Error, "betStartTime must be a Number");
+    });
+
+    it('It should throw if resultSetEndTime is null', () => {
+      input.resultSetEndTime = null;
+      assert.throws(() => new MultipleResultsEvent(input), Error, "resultSetEndTime must be a Number");
+    });
+
+    it('It should throw if resultSetEndTime is number', () => {
+      input.resultSetEndTime = "12";
+      assert.throws(() => new MultipleResultsEvent(input), Error, "resultSetEndTime must be a Number");
+    });
+
+    it('It should throw if resultSetEndTime is array', () => {
+      input.resultSetEndTime = [2];
+      assert.throws(() => new MultipleResultsEvent(input), Error, "resultSetEndTime must be a Number");
+    });
+
+    it('It should throw if resultSetEndTime is object', () => {
+      input.resultSetEndTime = {id:2};
+      assert.throws(() => new MultipleResultsEvent(input), Error, "resultSetEndTime must be a Number");
+    });
+  });
+
+  describe('format', () => {
+    let input;
+    const results = ['Invalid', '1', 'y'];
+
+    beforeEach(() => {
+      input = {
```

Comment:
can create a const at the of file. this input looks the same as the validate one.